nautobot/ipam/filters.py

```diff
@@ -323,7 +324,7 @@ def search_contains(self, queryset, name, value):
         return prefixes_queryset
 
     def generate_query_filter_present_in_vrf(self, value):
-        if isinstance(value, str):
+        if isinstance(value, (str, uuid.UUID)):
```

Comment:
Looks like a similar fix is needed in at least `IPAddressFilterSet`. Mind reviewing the rest of this file for any other cases?